article/Rougier-2017.md

```diff
@@ -0,0 +1,160 @@
+experimental implementation takes advantage of the fact that a set of cones
+seen from above using an orthographic projectionactually represents a Voronoi
+diagram (see [@Hoff:1999]). This also gives "for free" a rasterization of each
+Voronoi region provided each cone has a distinct color.
+
+Consequently, for all the figures in this article, we used instead
+the [QHull](http://www.qhull.org) library (through
+the [Scipy](https://scipy.github.io) Python package) for computing the Voronoi
+diagram together with the Voronoi regions (as a list of segments). We took care
+of adding extra points such that each cell is contained within a user-defined
+bounding box that is set to the dimension of the density image. Furthermore,
+each input image is resized (without interpolation) such that the mean pixel
+size of a voronoi cell is 500 (e.g. for 1000 stipples, the input image needs to
+be resized such it contains at least 500x1000 pixels).
+
+For computing the weighted centroid, we applied the definition proposed in the
+original paper over the discrete representation of the domain:
+
+$${\bf C}_i = \frac{\int_A {\bf x}\rho({\bf x})dA}{\int_A \rho({\bf x})}$$
+
+Each cell is rasterized (as a set of pixels) and the centroid is computed using
+the optimization proposed by the author that allow to avoid to compute the
+integrals over the whole set of pixels composing the Voronoi cell. As noted by
+the author, the precision of the method is directly related to the size of the
+Voronoi cell. Consequently, if the original density image is too small
+relatively to the number of stipples, there might be quality issues. We used a
+fixed number of iteration ($n=50$) instead of using the difference in the
+standard deviation of the area of the Voronoi regions as in the original paper
+since the definition of the rejection criterion was not clear in the original
+article and quite arbitrary.
+
+Last, we added a threshold parameter that allows to perform a pre-processing of
+the densty image. Any pixel with a gray level above the threshold is set to the
+threshold value before normalizing the density image. This was necessary for
+replicating some of the original images (see results section for further
+explanation).
+
+# Results
+
+We display only the output of our replication to be compared with the ones in
+the original articles. The climbing shoe (figure @fig:shoe), the corn plant
+(figure @fig:corn) and the large Peperomia plants (figure @fig:large-plant) are
+very similar to the images displayed in the original article. It it to be noted
+that we used different thresholds in order to obtain white areas similmar to
+the original images. Without such threshold there would be no reason to have
+white areas as for example in the leaves of figure @fig:large-plant.
+
+For the small Peperomia (figure @fig:small-plant), the output of our
+replication may be considered to be not exactly similar. If the hard edges are
```

Comment:
"may be considered to be not exactly similar" sounds a bit overly "careful" :)  perhaps "For the [...] the output of our replication is less similar".